docs/advanced/build_script.md

```diff
@@ -1,40 +1,40 @@
+It uses Python 2.7 programming language to drive all tests, so make sure to install Python 2.7 and include it in your system PATH. To compile mbed OS and tests, you will need one or more supported compilers installed on your system.
 
-To follow this short introduction you should already:
-* Know what mbed SDK is in general.
-* Know how to install Python 2.7, ARM target cross compilers.
-* You have C/C++ programming experience and at least willingness to learn a bit about Python. 
+To follow this introduction, you should:
+* Know what mbed OS is in general.
+* Know how to install Python 2.7 and ARM target cross compilers.
+* Have C/C++ programming experience and be willing to learn a bit about Python. 
 
 ## Test automation
-Currently our simple test framework allows users to run tests on their machines (hosts) in a fully automated manner. All you need to do is to prepare two configuration files.
+Our test framework allows users to run tests on their machines (hosts) in a fully automated manner. All you need to do is prepare two configuration files.
 
 ## Test automation limitations
-Note that for tests which require connected external peripherals, for example Ethernet, SD flash cards, external EEPROM tests, loops etc. you need to:
+Note that for tests that require connected external peripherals, such as Ethernet, SD flash cards, external EEPROM tests and loops, you need to:
 
 * Modify test source code to match components' pin names to actual mbed board pins where peripheral is connected or
 * Wire your board the same way test defines it.
 
 ## Prerequisites
 mbed test suite and build scripts are Python 2.7 applications and require Python 2.7 runtime environment and [setuptools](https://pythonhosted.org/an_example_pypi_project/setuptools.html) to install dependencies.
 
-What we need:
+You need:
 * Installed [Python 2.7](https://www.python.org/download/releases/2.7) programming language.
-* Installed [setuptools](https://pythonhosted.org/an_example_pypi_project/setuptools.html#installing-setuptools-and-easy-install) 
-* Optionally you can install [pip](https://pip.pypa.io/en/latest/installing.html) which is the PyPA recommended tool for installing Python packages from command line.
+* Installed [setuptools](https://pythonhosted.org/an_example_pypi_project/setuptools.html#installing-setuptools-and-easy-install).
+* Optionally, you can install [pip](https://pip.pypa.io/en/latest/installing.html), which is the PyPA recommended tool for installing Python packages from the command-line.
 
-mbed SDK in its repo root directory specifies ```setup.py``` file which holds information about all packages which are dependencies for it. Bear in mind only few simple steps are required to install all dependencies.
+mbed OS in its repo root directory specifies `setup.py` file, which holds information about all packages that are dependencies for it. Installing all dependencies requires a few steps.
```

Comment:
This should probably have a header explaining what happens here, and be edited a bit.